tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -43,21 +43,33 @@ object Discover {
   /** Find default clang compilation options. */
   def compileOptions(): Seq[String] = {
     val includes = {
-      val includedir =
-        Try(Process("llvm-config --includedir").lineStream_!.toSeq)
+      val llvmIncludeDir =
+        Try(Process("llvm-config --includedir").lineStream_!)
           .getOrElse(Seq.empty)
-      ("/usr/local/include" +: includedir).map(s => s"-I$s")
+
+      val includeDirs =
+        getenv("SCALANATIVE_INCLUDE_DIRS")
+          .map(_.split(" ").toSeq)
```

Comment:
A space in an env variable? That doesn't seem like the best choice. I would expect `java.io.File.pathSeparatorChar`.

Reply:
@sjrd I love it. Thanks!

Reply:
@sjrd technically it works because env is null terminated :) but I agree that it quite odd and can broke something. A lot of things.